README.md

```diff
@@ -1,10 +1,4 @@
-<p align='center'>
-    <img src='https://i.imgur.com/hq6SAZf.png' alt='elabftw logo' />
-</p>
-
-<p align='center'>
-<strong>A free, modern, flexible electronic lab notebook for researchers</strong>
-</p>
+![elabftw logo](https://i.imgur.com/hq6SAZf.png)
```

Comment:
Why did you change this part?

I want the logo centered and the sentence below it!